spec/system/candidate_interface/entering_details/candidate_entering_other_qualification_spec.rb

```diff
@@ -375,12 +372,4 @@ def and_i_delete_my_remaining_qualifications
     when_i_click_on_delete_my_first_qualification
     and_i_confirm_that_i_want_to_delete_my_additional_qualification
   end
-
-  def when_i_click_back_to_application_form
-    click_link 'Back to application'
```

Comment:
You can still keep these tests, but check for the details page. Something like:
```ruby
def when_i_click_back_to_my_details
  click_link 'Back to your details'
end

def then_i_see_the_your_details_page
  within '.govuk-heading-xl' do
    expect(page).to have_content 'Your details'
  end
end
```

Reply:
Cheers, have updated 👍🏼